File: server/src/test/java/com/afrtn/portal/TestResource.java
package com.afrtn.portal;

import io.quarkus.test.junit.QuarkusTest;
import org.junit.jupiter.api.Test;

import static io.restassured.RestAssured.given;
import static org.hamcrest.Matchers.equalToIgnoringCase;

@QuarkusTest
class TestResource {

    @Test
    void testIndexEndpoint() {
        given()
                .when().get("/")
                .then()
                .statusCode(200)
                .body("message", equalToIgnoringCase("Index"));
    }
}

File: server/src/test/kotlin/com/afrtn/portal/TestResource.kt
package com.afrtn.portal;

import io.quarkus.test.junit.QuarkusTest
import io.restassured.RestAssured.given
import org.junit.jupiter.api.Test
import org.hamcrest.Matchers.equalTo

@QuarkusTest
class TestResource {
    @Test
    fun testIndexEndpoint() {
        given()
            .`when`().get("/")
            .then()
            .statusCode(200)
            .body("message", equalTo("Index"));
    }
}